api/nomics.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import calendar
+import connexion
+
+from services.bitshares_websocket_client import client as bitshares_ws_client
+from services.bitshares_elasticsearch_client import client as bitshares_es_client
+from services.cache import cache
+import api.explorer
+import es_wrapper
+import config
+
+
+def info():
+    return {
+        "name": "Bitshares",
+        "description": "An exchange description of at least 1000 characters in plain text (no html)",
+        "location": "Anywhere",
+        "logo": "https://example.com/exchange-logo.png",
+        "website": "https://bitshares.org/",
+        "twitter": "example",
+        "capability": {
+            "markets": True,
+            "trades": True,
+            "tradesSocket": False,
+            "orders": False,
+            "ordersSocket": False,
+            "ordersSnapshot": True,
+            "candles": False
+        }
+    }
+
+
+def markets():
+    result = []
+    top_markets = bitshares_ws_client.request('database', 'get_top_markets', [100])
+
+    for market in top_markets:
+        result.append({
+            'id': market["base"] + "_" + market["quote"],
```

Comment:
From [the nomics API](https://github.com/nomics-crypto/nomics-platform/blob/master/doc/cryptocurrency-api-exchange-integration.md#markets---available-markets---required) it seems that they expect dash ('-') as delimiter, not underscore ('_').

Reply:
you are right, thanks. fixed at https://github.com/bitshares/bitshares-explorer-api/pull/64/commits/3ad8e65b6a97120446078c9b59efea4e1b02488a

Reply:
You are fast ;-)
I've been thinking about it, and I think it was not needed. Sorry.

Reply:
Yea, we actually give the IDs we want but in the sample response they use `-` so probably better.